File: sample apps/python/app/shared/guard.py
def validate_input(log_info, description, input_value):
    if input_value is None:
        raise ValueError("Invalid input")

    log_info("INPUT {}\n{}".format(description, input_value))


def validate_output(log_info, description, output_value):
    if output_value is None:
        raise ValueError("Invalid output")

    log_info("OUTPUT {}\n{}".format(description, output_value))

File: sample apps/JavaScript/src/common/guard.js
export function validateInput(logInfo, description, inputValue) {
    if (!inputValue)
        throw new Error('Invalid input');
    logInfo(`INPUT ${description}\n${inputValue.toString()}`);
}

export function validateOutput(logInfo, description, inputValue) {
    if (!inputValue)
        throw new Error('Invalid output');
    logInfo(`OUTPUT ${description}\n${inputValue.toString()}`);
}